invenio_pidstore/errors.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Errors for persistent identifiers."""
+
+
+class ResolverError(Exception):
+    """Base class for resolver errors."""
+
+    def __init__(self, pid_type, pid_value, *args, **kwargs):
+        """Initialize exception."""
+        self.pid_value = pid_type
+        self.pid_value = pid_value
+        super(ResolverError, self).__init__(*args, **kwargs)
+
+
+class PIDDoesNotExistsError(ResolverError):
```

Comment:
s/PIDDoesNotExistsError/PIDDoesNotExistError/g
Missed that one, do you want to release anyhow or do you want to fix it in the next release